packages/projects/src/components/ProjectForm.jsx

```diff
@@ -104,6 +111,13 @@ const value = (state, type) => {
   return state[type]
 }
 
+const getOwnerName = (ownerId, state) => {
```

Comment:
I know, but there is a call to `getOwnerName` in the `save` function where it doesn't have access to the `select` input, and I thought it would be best to get the owner name in a consistent way in both places so that it can't get out of sync.

The reason for the call inside of `save` is for the case where the user leaves the default option as-is. Then there's no `change` that fires, so I don't have an event other than `save` to trigger the setting of `ownerName`, and then in `save`, I don't know of a better way to get the text of the selected option, unless there's a way to just grab that specific DOM element.